### PR TITLE
Update php version recommendations

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: docs/requirements.md [hunk 1]
```diff
@@ -47,17 +47,19 @@ See our page on [choosing a CMS](/planning/cms.md) for more information about th
...
+| PHP 7.2 | compatible and **recommended** - but see note about resaving the SMTP password*| compatible and **recommended** but see note about resaving the SMTP password*|
+| PHP 7.1 | compatible | compatible |
+| PHP 7.0 | compatible but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in Dec 2018   | compatible but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in Dec 2018   |
+| PHP 5.6 | compatible but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in Dec 2018   | compatible but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in  Dec 2018  |
+| PHP 5.5 | compatible for patch versions prior to Feb 2019, but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in Dec 2018 |incompatible as of 5.10.0|
+| PHP 5.4 | **incompatible** | **incompatible** |
+| PHP 5.3 | **incompatible**  |  **incompatible** |
+
+* 7.2 proviso - 7.2 does not support mcrypt and if mcrypt is not installed the smtp password (if entered) will need to be re-saved once you update your php version to 7.2
```
Comment (seamuslee001) [inline docs/requirements.md]: can we make this as a !!! warning note or something

File: docs/requirements.md [hunk 2]
```diff
@@ -47,17 +47,19 @@ See our page on [choosing a CMS](/planning/cms.md) for more information about th
...
-| PHP 7.2 | **incompatible** | compatible and **recommended** with the proviso below*|
-| PHP 7.1 | **incompatible** | compatible and **recommended** |
-| PHP 7.0 | **incompatible** | compatible |
-| PHP 5.6 | compatible and **recommended** | compatible |
-| PHP 5.5 | compatible, but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in July 2016  |compatible until end of 2018, but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in July 2016 |
-| PHP 5.4 | compatible but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in Sept 2015 | **incompatible** |
-| PHP 5.3 | compatible but **not recommended** due to to [PHP end-of life](http://php.net/eol.php) in August 2014 |  **incompatible** |
-
-* 7.2 proviso - 7.2 does not support mcrypt and if mcrypt is not installed the smtp password (if entered) will not be encrypted in the database.
+| PHP 7.2 | compatible and **recommended** - but see note about resaving the SMTP password*| compatible and **recommended** but see note about resaving the SMTP password*|
```
Comment (seamuslee001) [inline docs/requirements.md]: should we include any note about php7.3?

Comment (eileenmcnaughton) [reply docs/requirements.md]: yeah good idea!

File: docs/requirements.md (hunk 1 above)
Comment (eileenmcnaughton) [reply docs/requirements.md]: I tried that - it now does as per further up on the page - not sure if it works though...

File: docs/requirements.md (hunk 2 above)
Comment (eileenmcnaughton) [reply docs/requirements.md]: updated

**Comment by MegaphoneJon** [comment]
lgtm

**Pull request merged**